config.yaml

```diff
@@ -230,8 +230,8 @@ postgresql_server:
 
   # postgresql.conf overrides
   config:
-    shared_buffers: '1024MB'
-    effective_cache_size: '1024MB'
+    #shared_buffers: '1024MB'
```

Comment:
probably add some comment here about the defaults

Reply:
It might even be better to remove them (rather than commenting them) and instead note that any variables compatible with <specific config file in our version of postgres> can be used here and will override our settings.